extensions/EXT/EXT_texture_compression_rgtc.txt

```diff
@@ -226,28 +253,71 @@ Dependencies on OpenGL 2.0 or ARB_texture_non_power_of_two
     spirit of the resolution of issue (3) of the EXT_texture_compression_s3tc
     specification.  See the "Implementation Note" section for more details.
 
+Interactions with the OpenGL ES 3.2 Specification
+
+    If implemented in OpenGL ES, replace the addition to Table 3.17
+    in Section 3.8.1 with an addition to Table 8.17 in Section 8.7:
+
+    Compressed Internal Format             Base      Block    Border  3D   Cube
+                                           Internal  Width x  Type    Tex  Map
+                                           Format    Height                Array
+                                                                           Tex
+    ---------------------------------      --------  -------  ------  ---  -----
+    COMPRESSED_RED_RGTC1_EXT               RED       4x4      unorm
+    COMPRESSED_SIGNED_RED_RGTC1_EXT        RED       4x4      snorm
+    COMPRESSED_RED_GREEN_RGTC2_EXT         RG        4x4      unorm
+    COMPRESSED_SIGNED_RED_GREEN_RGTC2_EXT  RG        4x4      snorm
+
+
+    Also, add to Section 8.4, Table 8.2: Valid combinations of format, type,
+    and sized internalFormat
+
+                           External
+                           Bytes
+    Format  Type           Per Pixel  Internal Format
+    ------  -------------  ---------  --------------------------------------
+    RED     UNSIGNED_BYTE  4          COMPRESSED_RED_RGTC1_EXT
+    RED     SIGNED_BYTE    4          COMPRESSED_SIGNED_RED_RGTC1_EXT
+    RG      UNSIGNED_BYTE  8          COMPRESSED_RED_GREEN_RGTC2_EXT
+    RG      SIGNED_BYTE    8          COMPRESSED_SIGNED_RED_GREEN_RGTC2_EXT
+
+
+    In OpenGL ES, queries to GL_NUM_COMPRESSED_TEXTURE_FORMATS and
+    GL_COMPRESSED_TEXTURE_FORMATS should return the RGTC formats.
+
+    In OpenGL ES, INVALID_OPERATION is generated by TexImage2D and TexStorage2D
+    if an RGTC format is used as the <internalFormat> parameter with a <type>
+    and <format> combination NOT listed:
+
+    InternalFormat                          Format      Type
+    ----------------------                  ----------  --------------
+    COMPRESSED_RED_RGTC1_EXT                RED         UNSIGNED_BYTE
+    COMPRESSED_SIGNED_RED_RGTC1_EXT         RED         SIGNED_BYTE
+    COMPRESSED_RED_GREEN_RGTC2_EXT          RG          UNSIGNED_BYTE
+    COMPRESSED_SIGNED_RED_GREEN_RGTC2_EXT   RG          SIGNED_BYTE
+
 Errors
 
     INVALID_ENUM is generated by CompressedTexImage1D
     or CompressedTexImage3D if <internalformat> is
-    COMPRESSED_LUMINANCE_LACT1_EXT, COMPRESSED_SIGNED_RED_RGTC1_EXT,
+    COMPRESSED_RED_RGTC1_EXT, COMPRESSED_SIGNED_RED_RGTC1_EXT,
```

Comment:
Parenthetically, I note that in addition to fixing the token (which should IMO be added to the change log), the *spelling* of the old COMPRESSED_LUMINANCE_LACT1_EXT token appears wrong in both EXT_texture_compression_{latc,rgtc}. Presumably that should be fixed as well?